src/assets/stylesheets/custom.scss

```diff
@@ -69,7 +69,21 @@ $textCenter: center;
 .db{ display: block; }
 .textCenter{ text-align: $textCenter; }
 .pointer{ cursor: pointer; }
-
+//user -gravatar
+ 
+ .user-avatar{
+     height: em(25);
+     width: em(25);
+     img{
+         width: $width100;
+         height: 100%;
+     }
+ }
+ .not-assigned-user-icon {
+   font-size: em(20);
+   border: 1px solid $color-pf-black-600;
```

Comment:
I have already created one mixin for border. Waiting to get it merged. This syntax will get changed